package/yast2-ruby-bindings.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Tue Oct 23 09:52:53 UTC 2018 - jreidinger@suse.com
+
+- Fix crash caused by reading external files in ASCII encoding
+  (bsc#1111367)
+- 3.3.0
```

Comment:
Version conflict, 3.3.0 has been already released in SLE15 with a different fix: https://github.com/yast/yast-ruby-bindings/blob/SLE-15-GA/package/yast2-ruby-bindings.changes#L61